eggbox_potential_sampler/sampling_data_view/tests/test_sampling_data_view.py

```diff
@@ -0,0 +1,84 @@
+import textwrap
+import unittest
+
+from chaco.lineplot import LinePlot
+
+from force_wfmanager.model.analysis_model import AnalysisModel
+
+from eggbox_potential_sampler.sampling_data_view import (
+    SamplingDataView)
+
+
+class TestSamplingDataView(unittest.TestCase):
+
+    def setUp(self):
+        self.analysis_model = AnalysisModel()
+        self.data_view = SamplingDataView(analysis_model=self.analysis_model)
+
+    def test_data_view(self):
+        # check the view composition (there are probably better ways of
+        # checking a traitsui.view.View)
+        expected_view = textwrap.dedent("""\
```

Comment:
The method `force-wfmanager.ui.ui_utils.model_info` could be useful here.

It's actually quite a useful way of splitting up `View` objects into each `Group` if ever needed

Reply:
:+1: Refactored using that module